packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -42,6 +47,7 @@ interface SelectPanelBaseProps {
   inputLabel?: string
   overlayProps?: Partial<OverlayProps>
   footer?: string | React.ReactElement
+  initialLoadingType?: InitialLoadingType
```

Comment:
This is the only new API addition, correct?

(non blocker) Aw, kinda sad that we have both `loading` and `(initial)LoadingType` as props, but I think we have to keep them for backward compatibility. I was hoping we could get a `loading`/`loader` enum type, but not a big deal.

Reply:
Ugh I know 😞 

This is the only API addition. There is one additional change too - the second parameter to `onFilterChange` (the `Event` object) is now nullable.

Reply:
We could use an enum for this, but I chose a class + constructor because I wanted to also include the `appearsInBody` attribute so you can do eg. `FilteredActionListLoadingTypes.bodySpinner.appearsInBody`. However, this particular callsite _does_ use an enum, `InitialLoadingType`, which is defined as `'spinner' | 'skeleton'`.